Update with new send event logs

andrewma2 opened the pull request:
<!--
**Your PR title must conform to the conventional commit spec!**

  <type>(<scope>)!: <description>

  * `type` = chore, enhancement, feat, fix, docs
  * `!` = OPTIONAL: signals a breaking change
  * `scope` = Optional when `type` is "chore" or "docs", available scopes https://github.com/vectordotdev/vector/blob/master/.github/semantic.yml#L20
  * `description` = short description of the change

Examples:

  * enhancement(file source): Add `sort` option to sort discovered files
  * feat(new source): Initial `statsd` source
  * fix(file source): Fix a bug discovering new files
  * chore(external docs): Clarify `batch_size` option
-->

Adding sending events log to aws_s3 (upgrading that and azure_blob log to INFO)

Keeping the same format as the existing azure_blob logs (bucket -> container, s3_key -> blob)

Tested with load tests in `dev-azure-koreacentral` + `dev-aws-us-west-2`.

Seeing INFO logs now

AWS
<img width="1693" alt="Screenshot 2024-04-04 at 10 48 45 AM" src="https://github.com/databricks/vector/assets/107439812/2a659681-2def-41eb-bd9a-140d72841686">


Azure
<img width="1710" alt="Screenshot 2024-04-02 at 6 13 47 PM" src="https://github.com/databricks/vector/assets/107439812/d2f95dee-9827-48a1-b08f-a1b206467cbc">

Comparison of logs between azure/aws (left/right respectively)

<img width="1697" alt="Screenshot 2024-04-02 at 6 21 29 PM" src="https://github.com/databricks/vector/assets/107439812/dd041ad4-e445-4a2c-9f76-f536c899e0d4">

andrewma2 committed:
Update with new send logs

anil-db reviewed (commented):
comparison looks good. but I think sample events you pasted under azure and aws are stale. can you paste the latest samples. 
title is also switched

andrewma2 commented:
> comparison looks good. but I think sample events you pasted under azure and aws are stale. can you paste the latest samples. title is also switched

Good catches, I have updated, thanks!
